src/components/DefineCompany/ListNameTranslations.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-card flat id="name-translations-list">
+    <!-- List Display Section -->
+    <div id="name-translations-header">
+
+      <!-- List Headers -->
+      <v-row class="name-translation-title list-item__subtitle" no-gutters>
+        <v-col>
+          <h3>Name Translations:</h3>
+        </v-col>
+      </v-row>
+
+      <!-- List Content -->
+      <v-row
+        class="names-translation-content"
+        v-for="(translation, index) in translationsList"
+        :key="index"
+        no-gutters>
+        <v-col class="text-truncate">
+         <span class="name-title">{{translation}}</span>
+        </v-col>
+
+        <!-- Actions Column -->
+        <v-col>
+          <div class="actions">
+            <span class="edit-action">
+              <v-btn
+                small
+                text
+                color="primary"
+                :disabled="isAddingNameTranslation"
+                @click="emitNameEdit(index)">
+                  <v-icon small>mdi-pencil</v-icon>
+                  <span>Edit</span>
+              </v-btn>
+            </span>
+            <!-- more actions menu -->
+            <span>
+              <v-menu offset-y>
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    text
+                    small
+                    v-on="on"
+                    color="primary"
+                    class="actions__more-actions__btn"
+                    :disabled="isAddingNameTranslation">
+                    <v-icon>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+                <v-list class="actions__more-actions">
+                  <v-list-item  @click="emitRemoveName(index)">
+                    <v-list-item-title color="primary"><v-icon>mdi-delete</v-icon>Remove</v-list-item-title>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </span>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Vue, Emit } from 'vue-property-decorator'
+
+// Dialogs
+import { ConfirmRemoveDialog } from '@/components/dialogs'
+
+@Component({
+  components: {
+    ConfirmRemoveDialog
```

Comment:
I don't see where this component is used...?

Reply:
It's copypasta error 😓. Removed in next commit!